pkg/controller/cluster/main-controller.go

```diff
@@ -489,8 +496,14 @@ func (c *Controller) syncHandler(key string) error {
 			if err != nil {
 				return err
 			}
+
+			// Probe if disco is configured at cluster level
+			discoSvcIP, err := mi.ProbeDisco(c.kubeClientSet)
+			if err != nil {
+				return err
+			}
```

Comment:
Do we need to probe here again, since this is the zone addition case meaning probing must have happened already